recipes/libcurl/all/conanfile.py

```diff
@@ -668,17 +639,12 @@ def package_info(self):
             self.cpp_info.components["curl"].libs = ["libcurl_imp"] if self.options.shared else ["libcurl"]
         else:
             self.cpp_info.components["curl"].libs = ["curl"]
-            if self.settings.os in ["Linux", "FreeBSD"]:
-                if self.options.with_libidn:
-                    self.cpp_info.components["curl"].libs.append("idn")
```

Comment:
No idn library is generated for this option, it's just a library used as a dependency, my geuss is that this never worked, but I have to double check where this comes from before merging